ledger/participant-integration-api/src/test/suite/scala/platform/store/CompletionFromTransactionSpec.scala

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.store
+
+import com.daml.ledger.offset.Offset
+import com.google.protobuf.timestamp.Timestamp
+import com.google.rpc.status.Status
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+import java.time.Instant
+
+class CompletionFromTransactionSpec extends AnyWordSpec with Matchers {
+
+  "CompletionFromTransaction" should {
+    "acceptedCompletion" in {
+      val completionStream = CompletionFromTransaction.acceptedCompletion(
+        Instant.EPOCH,
+        Offset.beforeBegin,
+        "commandId",
+        "transactionId",
+        "applicationId",
+        Some("submissionId"),
+      )
+
+      val checkpoint = completionStream.checkpoint.get
+      checkpoint.recordTime shouldBe Some(Timestamp(Instant.EPOCH))
+      checkpoint.offset shouldBe a[Some[_]]
```

Comment:
Couldn't come up with a better assertion, as it was always failing, so in the interest of time, I ignored this for now.

Reply:
There would be also more cases to test. However, the integration test coverage is going to increase with this PR: https://github.com/digital-asset/daml/pull/10868